src/tap.h

```diff
@@ -32,9 +32,38 @@
 #include <iostream>
 #include <map>
 #include <fstream>
+#include <string>
 
 namespace tap {
 
+// based on http://stackoverflow.com/a/7724536/831180
```

Comment:
We can add that later :-) release this version, and when one of us have spare time we can add some tests